archive.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+SHOPWARE_DIRECTORY=../../../../shopware
+
+VERSION_RAW="$(sed -n 's|<version>\(.*\)</version>|\1|p' ./FinSearchUnified/plugin.xml)"
+
+VERSION="$(echo -e "${VERSION_RAW}" | tr -d '[:space:]')"
+
+echo "Version: ${VERSION}"
+
+cp -rvf ./FinSearchUnified/ ${SHOPWARE_DIRECTORY}/tmp
+
+echo "Copying files ... "
+
+cd FinSearchUnified
```

Comment:
I guess this is the wrong directory?

Reply:
no, why? when you clone the repository you get the plugin directory inside it and since the archive.sh is in the root of the repository (project) I am doing a cd into the plugin directory to run the composer commands. Am I missing something here?

Reply:
The original idea was:
* copy FinSearchUnified into `/tmp/FinSearchUnified`
* `cd` into `/tmp/FinSearchUnified`
* run the archive command

Right now you are copying the plugin into shopware's tmp folder but don't do anything with it.